renderdoccmd/android/AndroidManifest.xml

```diff
@@ -9,11 +9,7 @@
   <uses-permission android:name="android.permission.INTERNET" />
 
   <application android:label="RenderDocCmd" android:hasCode="true">
-    <activity android:name=".Loader" android:label="RenderDocCmdLoader" android:configChanges="orientation|keyboardHidden">
-      <meta-data android:name="android.app.lib_name" android:value="renderdoccmd"/>
-    </activity>
-
-    <activity android:name="android.app.NativeActivity" android:label="RenderDocCmd" android:configChanges="orientation|keyboardHidden">
```

Comment:
False start.  One of the things I found with a search suggested that having two activities can cause the problem I was having.  I removed it, and the problem persisted.

Please forgive my ignorance, but is there a reason to keep it?  If so, I'm totally happy to put it back in.